### ISSUE TITLE
[2.2.4] Cannot build because of dependency w/ v28

### ISSUE THREAD

**Issue by varquiel** [PonnamKarthik/FlutterToast#47]
Everything works fine w/ 2.2.3, but since release of 2.2.4, build fails. Problem comes from the new dependency on v28.

I solved it by forcing 2.2.3 in pubspec.yaml, but is this something you can fix, or we have to make changes to our build.gradle (in that case, what exactly?)

Thanks,
Manuel.

**Comment by Icosa2050** [comment]
same here. thanks @varquiel. 2.2.3 saves the day.

**Comment by PonnamKarthik** [comment]
@varquiel Can you please post the complete error log

**Comment by zfha** [comment]
iOS is OK, but Android is wrong.
error log is
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Android resource linking failed
  Output:  /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/incremental/mergeDebugResources/merged.dir/values-v28/values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/incremental/mergeDebugResources/merged.dir/values-v28/values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/incremental/mergeDebugResources/merged.dir/values/values.xml:493: error: resource android:attr/fontVariationSettings not found.
  /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/incremental/mergeDebugResources/merged.dir/values/values.xml:493: error: resource android:attr/ttcIndex not found.
  error: failed linking references.

  Command: /Users/zhoufan/.gradle/caches/transforms-1/files-1.1/aapt2-3.2.1-4818971-osx.jar/ad516ebd2afd29632d3f28e608a635e0/aapt2-3.2.1-4818971-osx/aapt2 link -I\
          /Users/zhoufan/Library/Android/sdk/platforms/android-27/android.jar\
          --manifest\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/merged_manifests/debug/processDebugManifest/merged/AndroidManifest.xml\
          -o\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/processed_res/debug/processDebugResources/out/resources-debug.ap_\
          -R\
          @/Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/incremental/processDebugResources/resources-list-for-resources-debug.ap_.txt\
          --auto-add-overlay\
          --java\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/generated/not_namespaced_r_class_sources/debug/processDebugResources/r\
          --custom-package\
          com.example.bmatchflutter.host\
          -0\
          apk\
          --output-text-symbols\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/symbols/debug/R.txt\
          --no-version-vectors
  Daemon:  AAPT2 aapt2-3.2.1-4818971-osx Daemon #0
  Output:  /Users/zhoufan/.gradle/caches/transforms-1/files-1.1/appcompat-1.0.2.aar/a23fc78e2b3f48195a601b5628f3b479/res/values-v28/values-v28.xml:9:5-12:13: AAPT: error: resource android:attr/dialogCornerRadius not found.

  /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/incremental/mergeDebugResources/merged.dir/values-v28/values-v28.xml:11: AAPT: error: resource android:attr/dialogCornerRadius not found.

  /Users/zhoufan/.gradle/caches/transforms-1/files-1.1/design-27.1.1.aar/859338a072960e4b05fc6f6164803b93/res/values/values.xml:96:5-202: AAPT: error: resource android:attr/fontVariationSettings not found.

  /Users/zhoufan/.gradle/caches/transforms-1/files-1.1/design-27.1.1.aar/859338a072960e4b05fc6f6164803b93/res/values/values.xml:96:5-202: AAPT: error: resource android:attr/ttcIndex not found.

  error: failed linking references.
  Command: /Users/zhoufan/.gradle/caches/transforms-1/files-1.1/aapt2-3.2.1-4818971-osx.jar/ad516ebd2afd29632d3f28e608a635e0/aapt2-3.2.1-4818971-osx/aapt2 link -I\
          /Users/zhoufan/Library/Android/sdk/platforms/android-27/android.jar\
          --manifest\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/merged_manifests/debug/processDebugManifest/merged/AndroidManifest.xml\
          -o\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/processed_res/debug/processDebugResources/out/resources-debug.ap_\
          -R\
          @/Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/incremental/processDebugResources/resources-list-for-resources-debug.ap_.txt\
          --auto-add-overlay\
          --java\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/generated/not_namespaced_r_class_sources/debug/processDebugResources/r\
          --custom-package\
          com.example.bmatchflutter.host\
          -0\
          apk\
          --output-text-symbols\
          /Users/zhoufan/Documents/bmatch/bmatch-app/bmatch_flutter/build/host/intermediates/symbols/debug/R.txt\
          --no-version-vectors
  Daemon:  AAPT2 aapt2-3.2.1-4818971-osx Daemon #0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
Finished with error: Gradle task assembleDebug failed with exit code 1
```

**Comment by PonnamKarthik** [comment]
fixed v2.2.5

**Comment by varquiel** [comment]
Still same error, build of my app impossible w/ 2.2.5 (have to stay in 2.2.3 to use the plugin).

Logs:

C:\Users\Manuel\.gradle\caches\transforms-1\files-1.1\appcompat-1.0.2.aar\b73dc9c04e906fb263673f86a364dd0b\res\values-v28\values-v28.xml:9:5-12:13: AAPT: error: resource android:attr/dialogCornerRadius not found.
C:\Users\Manuel\Documents\scl_qualite\scl_qualite\build\app\intermediates\incremental\mergeDebugResources\merged.dir\values-v28\values-v28.xml:11: AAPT: error: resource android:attr/dialogCornerRadius not found.
C:\Users\Manuel\.gradle\caches\transforms-1\files-1.1\appcompat-1.0.2.aar\b73dc9c04e906fb263673f86a364dd0b\res\values\values.xml:1304:5-69: AAPT: error: resource android:attr/fontVariationSettings not found.
C:\Users\Manuel\.gradle\caches\transforms-1\files-1.1\appcompat-1.0.2.aar\b73dc9c04e906fb263673f86a364dd0b\res\values\values.xml:1304:5-69: AAPT: error: resource android:attr/ttcIndex not found.
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Failed to process resources, see aapt output above for details.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 1m 21s
Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
